src/patternfly/demos/LoginPage/examples/login-page-simple-example.hbs

```diff
@@ -12,20 +12,97 @@
     {{#> login-main}}
       {{#> login-box}}
         {{#> login-box-header}}
-          {{#> title titleType="h1" title--modifier="pf-m-3xl"}}
-            Login box title
+          {{#> title titleType="h1" title--modifier="pf-m-3xl pf-u-mb-md"}}
+            Sign in to your account
           {{/title}}
+          {{#> display display--type="inline-flex" display--modifier="pf-u-align-items-center"}}
+            {{#> dropdown id="dropdown-example-collapsed" pf-c-dropdown--HasToggleIcon="true"}}
+              English
+            {{/dropdown}}
+            <p class="pf-u-ml-md">Need an account? <a href="https://www.patternfly.org/">Sign up.</a></p>
+          {{/display}}
         {{/login-box-header}}
         {{#> login-box-body}}
-          Login box body. Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure labore ipsa nemo nostrum id culpa atque nulla expedita at! Explicabo aut minima alias. Officia magnam est at omnis iste eius?
+          {{#> form }}
+            {{#> form-group}}
+              {{#> form-label form-label--attribute='for="login-demo-form-username"' required="true"}}Username{{/form-label}}
+              {{#> form-control controlType="input" form-control--attribute='required input="true" type="text" id="login-demo-form-username" name="login-demo-form-username" aria-describedby="login-demo-username-helper"'}}{{/form-control}}
```

Comment:
The text "required" is showing up with the label text "Username".

Reply:
Yeah, this seems to be coming from the Form component itself: https://pf-next.com/components/Form/examples/

When that gets fixed this demo should as well.

Reply:
This PR now includes a fix for this. It turned out to be a Handlebars syntax issue within form-label.hbs that Matt Carleton confirmed.